lang/main.json

```diff
@@ -317,7 +322,9 @@
         "doNotShowWarningAgain": "Don't show this warning again",
         "doNotShowMessageAgain": "Don't show this message again",
         "permissionDenied": "Permission Denied",
-        "screenSharingPermissionDeniedError": "You have not granted permission to share your screen.",
+        "screenSharingFailedToInstall": "Oups! Your screen sharing extension failed to install.",
+        "screenSharingFailedToInstallTitle": "Screen sharing extension failed to install",
+        "screenSharingPermissionDeniedError": "Oups! Something went wrong with your screen sharing extension permissions. Please restart and try again.",
```

Comment:
Does "restart" make sense to you in the context of a web app? Does it mean refresh/reload? I see this phrase being used elsewhere so you probably shouldn't change it with this pr.